index.js

```diff
@@ -72,9 +72,11 @@ var browserslist = function (selections, opts) {
         selections = selections.split(/,\s*/);
     }
 
-    if ( opts.stats || process.env.BROWSERSLIST_STATS ) {
+    var statsFile;
+
+    var getStats = function () {
```

Comment:
Code become better but still too tricky. What do you think about this:

```js
var stats = opts.stats || process.env.BROWSERSLIST_STATS
if ( !stats ) {
  // search in files
}
if ( !stats ) {
  // show error
} else {
  // use custom stats
}
```

In this case you don’t need `getStats` function and could read code from top to bottom, without jumping in file.